src/Psl/Result/tryResultFrom.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Result;
+
+use Exception;
+
+/**
+ * Wrap the given callable result in a `Ok`, or `Failure` if the callable throws
+ * an `Exception`.
+ *
+ * @template     T
+ *
+ * @psalm-param  (callable(): T) $fun
+ *
+ * @psalm-return IResult<T>
+ */
+function tryResultFrom(callable $fun): IResult
```

Comment:
this doesn't follow PSL naming convention ( functions/variables should use `snake_case` casing ), and once renamed to `try_result_from`, it's a bit long.

I think `result` is redundant since we are inside the `Result` namespace.

`from` is also redundant, as I don't see us adding `try` in the future that might hold another meaning.

so I would go with either `Result\try`, or keep it `Result\wrap`.

but I think `Result\try` is better.

Reply:
Sadly `try` is a reserved keyword. Suggesting to use `try_from` which results in:

```php
$result = try_from($myAwesomeCallback);

$result->proceed($sendHttpSuccess, $failGracefully)
```


Looks good to me.

Reply:
from doesn't make sense in my opinion, let's keep `wrap`, as we are wrapping the result.